profile.html

```diff
@@ -98,9 +98,9 @@ <h3>About me</h3>
             </button>
           </div>
           <hr />
-          <div class="profile--settings">
+          <div class="profile__settings">
             <h3>About this App</h3>
-            <div class="darkmode--card">
+            <div class="darkmode__card">
               <p>Designed and Coded by newbie</p>
             </div>
```

Comment:
I am not sure why you choose `profile__settings` and `darkmode__card` as names as you did not add possibilities for the user to change the settings. I think maybe a name that resembles the content of this part would be better. :)

And I think that you do not need the extra `div` around your `p` element. Just add the class to the paragraph. :)

Reply:
Thank you again for the detailed review!